core/domain/stats_services.py

```diff
@@ -543,7 +542,7 @@ def get_exp_issues_from_model(exp_issues_model):
     """
     unresolved_issues = []
     for unresolved_issue_dict in exp_issues_model.unresolved_issues:
-        _migrate_to_latest_issue_schema(copy.deepcopy(unresolved_issue_dict))
+        _migrate_to_latest_issue_schema(unresolved_issue_dict)
```

Comment:
Apply the migration to a copy, not the actual dict.

```python
unresolved_issue_dict_copy = copy.copy(unresolved_issue_dict)
_migrate_to_latest_issue_schema(unresolved_issue_dict_copy)
unresolved_issues.append(
    stats_domain.ExplorationIssue.from_dict(unresolved_issue_dict_copy))
```

Reply:
done